### PR TITLE
[TIMOB-25850] : iOS SearchBar doesn't show Bookmark button when set on creation in TableView

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-25850

### CODE REVIEW

**Review by hansemannn — CHANGES_REQUESTED** [review]
Please loop through the proxy keys instead of checking all manually.

File: iphone/Classes/TiUISearchBarProxy.m [hunk 1]
```diff
@@ -89,6 +89,15 @@ - (void)setSearchBar:(UISearchBar *)searchBar
 
   // In UISearchController searchbar is readonly. We have to replace that search bar with existing search bar of proxy.
   [(TiUISearchBar *)[self view] setSearchBar:searchBar];
+
+  // Set search bar properties to new search bar
+  NSDictionary *properties = [self allProperties];
+  for (NSString *key in properties.allKeys) {
```
Comment (hansemannn) [inline iphone/Classes/TiUISearchBarProxy.m]: I assume `[self allProperties]` (`dynprops`) cannot be `nil`, otherwise the app could crash here.

**Review by hansemannn — APPROVED** [review]

**Comment by hansemannn** [comment]
@vijaysingh-axway Do we have tests for all properties of the search-bar already? If not, please add them. We should get too easy with setting the `no tests` label, otherwise it might get removed again.

**Comment by vijaysingh-axway** [comment]
@hansemannn In TIUISearchBar, getter is not implemented properly. It simply returns the value which is set in proxy. Ideally it should return value by fetching from searchbar. Currently whatever you set the value of property, it will return same even though that value is not passed to searhbar. So I am thinking to create a ticket for that, in which we can add all these property in unit tests as well. Thats why I have not added unit test in this ticket. Lets discuss on this.

**Comment by hansemannn** [comment]
Like it, please do so! Maybe for the 7.3.0 timeline then.

**Comment by hansemannn** [comment]
Closing in favor of #9947

**Pull request closed without merging**